firefox_ui_tests/remote/security/manifest.ini

```diff
@@ -1,3 +1,5 @@
+[test_dv_certificate.py]
+[test_mixed_script_content_blocking.py]
```

Comment:
I think you didn't branch off from master but from your former integration branch.